applications/dashboard/models/class.activitymodel.php

```diff
@@ -879,17 +883,24 @@ public function sendNotification($ActivityID, $Story = '', $Force = false) {
                     // Only send if the user is not banned
                     if (!val('Banned', $User)) {
                         $Email->send();
+                        $Emailed = self::SENT_OK;
```

Comment:
I think that not sending an email to a banned user means that the result should be skipped. We want to be able to be sure that any email with SENT_OK means an actual email was sent.